libs/quasselc/Makefile

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=quasselc
+
+# quasselc upstream doesn't release versions (at least, at the moment),
+# so use 0 here (in case they decide to later) for now.
+# I'm not sure if there's an openwrt convention here.
+PKG_VERSION:=0
+PKG_SOURCE_VERSION:=fcd966966924e3d9af0954db56117e2f48767ea1
+PKG_RELEASE:=1.$(PKG_SOURCE_VERSION)
+
+PKG_LICENSE:=GPL-3.0+
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/phhusson/QuasselC
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+
+PKG_MAINTAINER:=Ben Rosser <rosser.bjr@gmail.com>
+
+PKG_BUILD_DEPENDS:=glib2
```

Comment:
Build depends are for dependencies at build-time.  This relationship should already be covered by the later `DEPENDS := +glib2` and can be dropped here.